Serializable/Source/Extensions.swift

```diff
@@ -48,6 +48,15 @@ extension UIColor:HexInitializable {
             return nil
         }
         
+        let regexString = "[0-9A-F]{6}"
+        
+        let regex = try! NSRegularExpression(pattern: regexString, options: NSRegularExpressionOptions(rawValue: 0))
```

Comment:
We need to create the regex just once. It's performance heavy to create it every time we serialize a UIColor.